tests/Unit/Core/Cart/Adding/CartRule/AddRuleTest.php

```diff
@@ -76,6 +76,7 @@ public function testCartRuleValidity(
             $this->cart->addCartRule($cartRule->id);
         }
         $this->assertEquals($shouldRulesBeApplied, $result);
+        $this->assertTrue( \CartRule::haveCartRuleToday(0));
```

Comment:
Can you remove the space and add namespace to rerun travis build (blocked in travis maybe due to sha1 problem.. :/)